ngi_reports/stockholm/project_summary.py

```diff
@@ -36,6 +37,13 @@ def __init__(self, config, LOG, working_dir, **kwargs):
                 self.LOG.error("No project name found - please specify using '--project'")
                 raise KeyError("No project name found - please specify using '--project'")
         
+        ## Check and exit if signature not provided
+        if not kwargs.get('signature'):
+            self.LOG.error("It is required to provide Signature/Name generating 'project_summary' report")
```

Comment:
Tack for the info @ewels , will look into it and try to implement in the future PR :+1: